development/tools/cmd/gcrcleaner/README.md

```diff
@@ -0,0 +1,46 @@
+# GCR cleaner
+
+## Overview
+
+This command finds and removes old GCR images created by jobs in the Google Cloud Platform (GCP) project.
+
+There are three conditions used to find images for removal:
+- The repository name pattern is not on the ignored list.
+- The `creationTimestamp` value of the images, that is used to find addresses, exists at least for a preconfigured number of hours.
```

Comment:
```suggestion
- The `creationTimestamp` value of the images, which is used to find addresses, exists at least for a preconfigured number of hours.
```
You can't use `that` in a non-defining clause (the one separated with commas). In such a context, you must use `which`. See [this for reference](https://dictionary.cambridge.org/grammar/british-grammar/relative-clauses-defining-and-non-defining).